controllers/persistentvolume/reconcile_test.go

```diff
@@ -68,8 +75,11 @@ func TestEnsureExpansionSecret(t *testing.T) {
 
 		testSC := &storagev1.StorageClass{}
 		mockStorageClass.DeepCopyInto(testSC)
-		testSC.Parameters[csiExpansionSecretName] = c.secretName
-		testSC.Parameters[csiExpansionSecretNamespace] = c.secretName
+		if c.allowExpansion {
+			testSC.Parameters[csiExpansionSecretName] = c.secretName
+			testSC.Parameters[csiExpansionSecretNamespace] = c.secretName
```

Comment:
Should not it set namespace as a value?

Reply:
Yes correct, its a bug in current code will fix it